org.opent2t.sample.lamp.superpopular/com.smartthings.lightbulb/js/thingTranslator.js

```diff
@@ -248,7 +249,6 @@ function resourceSchemaToProviderSchema(resourceId, resourceSchema) {
             break;
         case 'colourRGB':
             var HSVColor = RGBtoHSV(resourceSchema.rgbvalue);
-            console.log(HSVColor);
```

Comment:
Doesnt have access to the logger in this scope. Will need to figure out/refactor a bit so for now no.

---
In reply to: [103830317](https://github.com/openT2T/translators/pull/188#discussion_r103830317) [](ancestors = 103830317)